README.md

```diff
@@ -207,6 +207,45 @@ There is available REST API for AMOC where we can:
 * check status of nodes in AMOC cluster
 
 API is described [here](REST_API_DOCS.md). You can also get an access to REST API by running AMOC and go to `(address:port)/api-docs`.
-### Docker
 
-TODO
+## Docker
+
+Building local Docker image may be started with:
```

Comment:
`To build a Docker image with Amoc, run the following command from the root of the repository:` - what do you think about it?